src/testing/basic_test.rs

```diff
@@ -80,6 +81,12 @@ mod tests {
             type InstanceState = TestInstanceState;
             type Membership = GeneralStaticCommittee<TestTypes, Self::SignatureKey>;
             type BuilderSignatureKey = BuilderKey;
+            type Base = StaticVersion<0, 1>;
+            type Upgrade = StaticVersion<0, 2>;
+            const UPGRADE_HASH: [u8; 32] = [
+                1, 0, 1, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 1, 0, 0,
```

Comment:
I get that this is an unused initialized associated constant for a test, but out of curiosity, what is the significance of this sequence of bytes? Increasing distance in order to debug offset memory in the data segment?

Reply:
Haha, way simpler: it's meaningless. It could as easily be all 0's